src/main/java/sg/ncl/MainController.java

```diff
@@ -2547,6 +2549,10 @@ public String teamAppUnderReview() {
         return "team_application_under_review";
     }
 
+    // model attribute name come from /login
+    @RequestMapping("/email_checklist")
+    public String emailChecklist(@ModelAttribute("statuschecklist") String status) {return "email_checklist";}
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this method and declare a constant for this value. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS3400)